### PR TITLE
rgp: various site page text edits

### PR DESCRIPTION
**Pull request by yufengwng**
Lumping a bunch of text edits together:
* DDP-6408 (still waiting for updated PDF)
* DDP-6410
* DDP-6411
* DDP-6412
* DDP-6414
* DDP-6416 (confirmed, we do want linking to sharing-guide pdf)

### CODE REVIEW

File: ddp-workspace/angular.json [hunk 1]
```diff
@@ -1987,7 +1987,7 @@
               "projects/ddp-rgp/src/assets",
               {
                 "glob": "*.pdf",
-                "input": "projects/ddp-rgp/src/pdf/",
+                "input": "projects/ddp-rgp/src/assets/pdf/",
```
Comment (yufengwng) [inline ddp-workspace/angular.json:1990]: Looks like this is looking at the wrong folder?

File: ddp-workspace/projects/ddp-rgp/src/assets/i18n/en.json [hunk 2]
```diff
@@ -533,31 +540,38 @@
...
                     "question": "Who in my family will get their genome sequenced?",
                     "answers": [
-                        "It depends. Typically, we perform genome sequencing on the affected individual(s) in the family and their biological parents (when available). We typically only perform targeted testing for specific variants of interest in unaffected family members."
+                        "It depends. Typically, we perform genome sequencing on the affected individual(s) in the family and their biological parents, when possible. We typically only perform targeted testing for specific variants of interest in unaffected family members."
+                    ]
+                },
+                {
+                    "question": "What medical records are needed?",
+                    "answers": [
+                        "We require medical records from the affected family member(s). For more information about the types of records that are needed, please see our <a href=\"/sharing-guide.pdf\" target=\"_blank\" class=\"Link\">Guide to Sharing Health Information</a>."
```
Comment (yufengwng) [inline ddp-workspace/projects/ddp-rgp/src/assets/i18n/en.json:555]: Somehow this link to the sharing-guide PDF in the FAQ page is not working for me when I run the app locally.

File: ddp-workspace/angular.json (hunk 1 above)
Comment (aleksey-kuznetsov) [reply ddp-workspace/angular.json:1990]: Right, now it's correct

File: ddp-workspace/projects/ddp-rgp/src/assets/i18n/en.json (hunk 2 above)
Comment (aleksey-kuznetsov) [reply ddp-workspace/projects/ddp-rgp/src/assets/i18n/en.json:555]: Interesting, it doesn't work for me either

Comment (yufengwng) [reply ddp-workspace/projects/ddp-rgp/src/assets/i18n/en.json:555]: For now, I think I'm going to merge this in as-in and investigate/fix it in a separate PR.

**Comment by yufengwng** [comment]
Alright, since it's just text edits here I'm going to push this up now and start another batch of edits.

**Pull request merged**